src/redux/modules/OutgoingReducer.jsx

```diff
@@ -12,44 +18,62 @@ export const getIsLoading = state => state.outgoings.isLoading;
 // Async Action Creators
 // ------------------------------------
 
-const doLoadOutgoings = () => (dispatch, getState) => {
+export function doLoadOutgoings() {
```

Comment:
Warum exportierst du diese Funktion hier und in den actions? Vielleicht müssen wir uns da noch Besprechen, auch wie wir bezüglich Testing damit umgehen wollen...